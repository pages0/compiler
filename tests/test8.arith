if 5 <= 4  then 5 <= 4  else (1 + 1) * (8 / 4)