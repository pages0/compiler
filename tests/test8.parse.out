( if ( <= 5 4 ) ( <= 5 4 ) ( * ( + 1 1 ) ( / 8 4 ) ) )